web/angular/partials/linea.html

```diff
@@ -30,11 +37,15 @@
         <i class="fa fa-briefcase"></i>
       {{customization.role ? customization.role : 'Rol'}}</a>
     </li>
+
+
     <li ng-click="filterLine('territory')" class="" ng-class="filter === 'territory' ? 'active' : '' ">
       <a id="ordenCargo" href="" >
         <i class="fa fa-flag"></i>
-      {{customization.territory ? customization.territory : 'Territorio'}}</a>
+      {{customization.territory ? customization.territory : 'Organización'}}</a>
```

Comment:
Este cambio es innecesario, por lo que se ve en el código se puede modificar en cargografias-locdata.json sin modificarlo aquí, además mientras no funcione no incluiría esto.
```suggestion
      {{customization.territory ? customization.territory : 'Territorio'}}</a>
```